modules/home/api.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace Elementor\Modules\Home;
+
+class API {
+
+	const HOME_SCREEN_DATA_URL = 'https://assets.stg.elementor.red/home-screen/v1/home-screen.json';
+
+	public static function get_home_screen_items( $force_request = false ): array {
```

Comment:
Why all the methods in this file are static?

Reply:
I have followed the implementation of the /notifications module as close as possible. Yakir had used all static methods. I assume he has done that to avoid additional instantiations of the class. I am happy to leave it as is, but 
I can change it as well. 

Another question: we possibly could refactor the code further and create a base for some of these methods. I just don't know where to create the base and how to name it. Wdyt?

Possibly we could do a huddle with you (and Louis?) just now if necessary.

Reply:
@davseve Maybe we can leave this question open, and check with @louiswol94 ?

Reply:
I believe @hein-obox did very well to mimic the current behavior. But there is now a lot of duplicated code between apps/notifications/home-screen. Perhaps we can open a task under Engineering Excellence to refactor this code.